scripts/publish-mobile-client.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Usage: ./scripts/publish-mobile-client.sh $COMMIT_SHA $NPM_TOKEN
+
+set -euo pipefail
+
+# Process args
+commit_sha_short="${1:0:7}"
+npm_token="${2:-}"
+
+package_name="$(node -p "require('./package.json').name")"
+
+version=$(npm show "$package_name" version)
+# Bump minor version
+a=( ${version//./ } )
+((a[2]++))
+new_version="${a[0]}.${a[1]}.${a[2]}"
+
+# Add npm token if provided
+if [ -n "$npm_token" ]; then
+  echo "//registry.npmjs.org/:_authToken=$npm_token" > ~/.npmrc
+fi
+
+rm README.md
```

Comment:
Why remove the README?

Reply:
Not sure, I copied this from the previous script.
@cmcewen do you know why we did this?

Reply:
I'm guessing it's why we see this error here: https://www.npmjs.com/package/@celo/client

Reply:
Yeah that's just because the readme wasn't up to date and is for the blockchain - thought we should make it clear that this is the mobile build

Reply:
Agreed, I've added a TODO for now.